evap/rewards/tests/test_views.py

```diff
@@ -119,11 +119,27 @@ class TestEventsView(WebTestStaffModeWith200Check):
     url = reverse("rewards:reward_point_redemption_events")
 
     @classmethod
-    def setUpTestData(cls):
+    def setUpTestData(cls) -> None:
         cls.test_users = [make_manager()]
```

Comment:
Can we do this so that we can use `self.manager` in the test?
```suggestion
        cls.manager = make_manager()
        cls.test_users = [cls.manager]
```